.env

```
@@ -0,0 +1,2 @@
+EXPO_BASE_URL = http://192.168.1.4:2000
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)